subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -984,10 +986,8 @@ int ull_central_iso_cis_offset_get(uint16_t cis_handle,
 	 *       `ull_conn_llcp()` is called before `ull_ref_inc()` hence we do not need to use
 	 *       `ull_conn_event_counter()`.
 	 */
-	cis->central.instant = conn->lll.event_counter + conn->lll.latency_prepare +
-			       conn->llcp.prep.lazy + 4U;
-
-	*conn_event_count = cis->central.instant;
+	*conn_event_count = conn->lll.event_counter + conn->lll.latency_prepare +
+			    conn->llcp.prep.lazy + CIS_CREATE_INSTANT_DELTA;
```

Comment:
The comment above seems misplaced now. Some of it should "belong to" the definition of `CIS_CREATE_INSTANT_DELTA`.